examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -470,15 +495,29 @@ func extractNonPointerInterfaces(comments []string) (bool, error) {
 	if len(values) == 0 {
 		return false, nil
 	}
-	result := values[0] == "true"
+	result := values[0] == "true" || values[0] == ""
```

Comment:
Our client-gen recently moved to accept `""` as true. IMO this is what one expects. Let me factor that comparison out. Then it's more explicit.